Assets/MRTK/Services/InputSystem/GazeProvider.cs

```diff
@@ -26,6 +26,10 @@ public class GazeProvider :
 
         private const float MovementThreshold = 0.01f;
 
+        [SerializeField]
+        [Tooltip("If true, the gaze cursor will stay locked on the object when the pointer's focus is locked, otherwise it will continue following the head's direction")]
+        private bool lockCursorWhenFocusLocked = true;
```

Comment:
Is there a way to set this property from code? Looks like it's a private serialized property with no getter / setter, but maybe I missed something? Following MRTK principle [Support configuring both in editor and a runtime](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#support-configuring-components-both-in-editor-and-at-run-time)

Reply:
Since this value is only used on startup when the pointer is created, I think we should encourage people to use the pointer's property at runtime (via `GazeProvider.GazePointer.IsTargetPositionLockedOnFocusLock`) instead of exposing the same concept twice.

Reply:
@RogPodge might be worth adding a note to that effect in the docs somewhere, if that's the path we go